### PR TITLE
Components composition

### PR DESCRIPTION
**Pull request by michaelamponsah**
On this feature branch, the following have been implemented

- [x] Composed the required components to build the layout required.

calc-layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
       <h1>Math Magicians</h1>
+      <Calculator />
```
Comment (mckent05) [inline src/App.js:8]: - According to the project requirements, this component should be class-based, please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @michaelamponsah, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job using the right workflow for this milestone 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (michaelamponsah) [reply src/App.js]: @mckent05, 
Many thanks! I might have missed that.

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @michaelamponsah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

## Highlights

✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**